src/index.js

```diff
@@ -56,7 +56,12 @@ const cloneWithChildren = (node, children, isRootEl, level) => {
 };
 
 const validateTree = (node) => {
-  if (typeof node === 'string' || isAtomComponent(node)) {
+  if (
+    node === null ||
```

Comment:
nit: checking for node == null will take care of both the null and undefined checks

Reply:
Updated.

Reply:
it should be `==`, not `===` 😅  Can you fix this and I'll merge it after?

Reply:
Done